xontrib_term_integrations/iterm2.py

```diff
@@ -29,3 +33,5 @@ def onpostinit(**__):
 
 
 XSH.env["PROMPT"] = ShellIntegrationPrompt(XSH.env)
+if not _skip_alias:
+    XSH.aliases["print_link"] = utils.write_osc_hyperlink_alias
```

Comment:
instead of defining it in three modules, can this be put in the module importing others (the entrypoint) ?

Reply:
I prefer a DRY solution , can this be placed in its own module and be imported in the loader?

Reply:
But your suggestion isn't DRY, it trades one form of repetition for another while losing alias colocation: you'll have to repeat loading `XONTRIB_TERM_INTEGRATIONS_SKIP_ALIAS` and its conditional which each term will have to load to set unique user var aliases

Reply:
yes but loading will be only one time than 3 repeats or potentially any other terminals

Reply:
it will be 4 times: 3 times in each terminal file (for user vars aliases), and 1 time in your extra module

Reply:
why not move all aliases to one file then. I don't see any variations in the aliases. Also this PR changes in all three files seems excess.

Reply:
> why not move all aliases to one file then.

because it doesn't solve any issue. Then you'd also have to repeat which-terminal check in this one file because...

> I don't see any variations in the aliases

I've mentioned several times that user var alias name is unique

Reply:
I don't see any issue in your code. I just want to minimal change possible. I will work on this PR myself later.